### PR TITLE
Fix BMO Local image building var naming

### PR DESCRIPTION
**Pull request by kashifest**

### CODE REVIEW

**Review by lentzi90 — COMMENTED** [review]
/lgtm
Explanation for why this is needed: We used a different variable name [here](https://github.com/metal3-io/metal3-dev-env/blob/main/03_launch_mgmt_cluster.sh#L40).

File: lib/common.sh [hunk 1]
```diff
@@ -195,7 +195,7 @@ if [ "${BUILD_IPAM_LOCALLY}" == "true" ]; then
   export IPAM_LOCAL_IMAGE="${IPAMPATH}"
 fi
 if [ "${BUILD_BMO_LOCALLY}" == "true" ]; then
-  export BMO_LOCAL_IMAGE="${BMOPATH}"
+  export BAREMETAL_OPERATOR_LOCAL_IMAGE="${BMOPATH}"
```
Comment (furkatgofurov7) [inline lib/common.sh:198]: Where is this `BMO_LOCAL_IMAGE` var is coming from

**Pull request merged**